data/rdm/manufacturer_pids.proto

```diff
@@ -2103,7 +2103,7 @@ manufacturer {
         }
         label {
           value: 1
-          label: "ArtNet"
+          label: "Art-Net"
```

Comment:
These also want updating here, which is where this data is sourced from if you're feeling keen: https://github.com/OpenLightingProject/rdm-app/blob/master/data/pid_data.py#L719